src/styles/utils/variables.scss

```diff
@@ -0,0 +1,23 @@
+$roboto-font-family: 'Roboto', sans-serif;
+$font-weight-medium: 500;
+$font-weight-regular: 400;
+$font-size-small: 10px;
+$font-size-medium: 12px;
+$font-size-large: 16px;
+$font-size-button: 14px;
+$line-height-small: 14px;
+$line-height-medium: 18px;
+$line-height-large: 18px;
+$line-height-button: 16px;
+$font-size-bold: 14px;
+$line-height-bold: 16px;
+$font-weight-bold: 700;
+$font-style-normal: normal;
+$color-bacground: white;
+$color-card-border: #f3f3f3;
+$color-main: #060b35;
+$color-secondary: #616070;
+$color-button-background: #00acdc;
+$color-button-text: white;
+$color-button-background-hoover: white;
```

Comment:
There is a typo in the variable name `$color-button-background-hoover`. It should be `$color-button-background-hover` to maintain consistency with the naming convention.